ReactWindows/Playground.Net46/PlaygroundBootstrapper.cs

```diff
@@ -0,0 +1,59 @@
+﻿using ReactNative;
+using System;
+using System.Windows;
+using System.Windows.Navigation;
+
+namespace Playground.Net46
+{
+    public sealed class PlaygroundBootstrapper : ReactNativeBootstrapper
```

Comment:
I have removed all bootstrapper related code from the Playground.Net46 and have gone with a more minimalist approach similar to Playground